web-gui/src/app/page-components/task-detail/task-detail.component.ts

```diff
@@ -71,6 +72,7 @@ export class TaskDetailComponent implements OnInit {
         if (submissions.length == 0) {
           this.status = <boolean>null
         } else {
+          this.pending = submissions.reverse()[0].results?.length == 0
```

Comment:
Since the existing list is reversed here, the submissions are displayed incorrectly.

Reply:
Hmm, I think I got used to have immutable states. Should just use `submissions[submissions.length - 1]` to access the last element.

However, am I right with the assumption that a submission is "pending" if the results array of a submission is empty?

Reply:
Yes, your assumption is correct.

Reply:
Just noticed that we have a done attribute. If used correctly is should be exactly the information that I require.

```json
[{
    "submissionTime": 1605484800000,
    "done": false,
    "id": 19198,
    "results": []
}]
```

Reply:
I have tried to submit the sql result `SELECT 1;` for the only existing task on the test server.
Waited about 15 Min without results. This is quite nice to test this feature, but does it suppose to work like this?

Reply:
yes, this is because no Cheker configuration was defined there.